src/repositories/prisma/prisma-segments-repository.ts

```diff
@@ -1,27 +1,39 @@
-import { Prisma, Segment } from "@prisma/client";
-import { SegmentsRepository } from "../segments-repository";
-import { prisma } from "@/lib/prisma";
+import { Prisma, Segment } from '@prisma/client'
+import { SegmentsRepository } from '../segments-repository'
+import { prisma } from '@/lib/prisma'
 
 export class PrismaSegmentsRepository implements SegmentsRepository {
-  async findManyListIds(ids: string[]): Promise<{ id: string; name: string; }[]> {
+  async findManyListIds(
+    ids: string[],
+  ): Promise<{ id: string; name: string }[]> {
```

Comment:
nao deveria ser assim?
```suggestion
  ): Promise<Segment[]> {
```